### DISCUSSION TITLE
[bug] scoop install error

### DISCUSSION THREAD

**Discussion by anglehome** [ScoopInstaller/Scoop#5120] [General]
![1](https://user-images.githubusercontent.com/48746111/179750984-eaf487e1-4de8-43ba-b5fe-5cd3044b119b.png)

![1](https://user-images.githubusercontent.com/48746111/179750951-7ac110e0-7662-46f3-870d-bdbb4649c624.png)

**Comment by niheaven** [comment]
What's your OS version and build number?

**Comment by anglehome** [comment]
> What's your OS version and build number?

window11   21H2

**Comment by niheaven** [comment]
This may due to unstable network access to github.com in mainland China. Consider using a proxy or some service e.g. fastgit.

**Comment by anglehome** [comment]
> This may due to unstable network access to github.com in mainland China. Consider using a proxy or some service e.g. fastgit.
i used proxy also can‘t install...

**Comment by niheaven** [comment]
@anglehome Have you tried reinstall scoop? And please give some feedback in installation and updating process.

**Comment by pheotry** [comment]
I hava the same problem.
How do you solve it ?
